website/docs/r/private_link_service.html.markdown

```diff
@@ -0,0 +1,163 @@
+
+  frontend_ip_configuration {
+    name                 = azurerm_public_ip.example.name
+    public_ip_address_id = azurerm_public_ip.example.id
+  }
+}
+
+resource "azurerm_private_link_service" "example" {
+  name                = "example-pls"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  fqdns               = ["testFqdns"]
+
+  nat_ip_configuration {
+    name                         = "primaryIpConfiguration"
+    subnet_id                    = azurerm_subnet.example.id
+    private_ip_address           = "10.5.1.17"
+    private_ip_address_version   = "IPv4"
+    private_ip_allocation_method = "Static"
+  }
+
+  load_balancer_frontend_ip_configuration_ids = [
+    id = azurerm_lb.example.frontend_ip_configuration.0.id
+  ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the private link service. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which the private link service resides. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `auto_approval_subscription_ids` - (Optional) A list of subscription globally unique identifiers(GUID) that will be automatically be able to use this service.
+
+* `nat_ip_configuration` - (Required) One or more `nat_ip_configuration` blocks as defined below.
+
+* `load_balancer_frontend_ip_configuration_ids` - (Required) A list of `Standard` Load Balancer resource ids to direct the service network traffic toward.
+
+* `private_endpoint_connection` - (Optional) One or more `private_endpoint_connection` blocks as defined below.
+
+* `visibility_subscription_ids` - (Optional) A list of subscription globally unique identifiers(GUID) that will be able to see this service. If left undefined all Azure subscriptions will be able to see this service.
+
+* `tags` - (Optional) A mapping of tags to assign to the resource. Changing this forces a new resource to be created.
+
+---
+
+The `nat_ip_configuration` block supports the following:
+
+* `name` - (Required) The name of private link service ip configuration.
+
+* `primary` - (Optional) If the `ip_configuration` is the primary ip configuration or not. Defaults to `true`.
+
+* `private_ip_address` - (Optional) The private IP address of the IP configuration.
+
+* `private_ip_allocation_method` - (Optional) The private IP address allocation method, supported values are `Static` and `Dynamic`. Defaults to `Dynamic`.
+
+* `subnet_id` - (Required) The resource ID of the subnet to be used by the service.
+
+* `private_ip_address_version` - (Optional) The ip address version of the `ip_configuration`, supported values are `IPv4` or `IPv6`. Defaults to `IPv4`.
```

Comment:
If only IPv4 is supported (and thats allt he schema allows) why list ipv6?

Reply:
Fixed.